### PR TITLE
Protolathe Printing and R&D technology 

### PR DESCRIPTION
**Pull request by SethLafuente**
## About the PR 

Added more items into the protolathe and added more technology trees to the R&D Console

**Screenshots**

image
Content Client_5PlAi49Loe

**Changelog**

:cl: Seth
- add: Added new technologies and recipes to the lathe.

### CODE REVIEW

**Comment by metalgearsloth** [comment]
DNM until submodule commit is removed

**Comment by SweptWasTaken** [comment]
> 
> 
> DNM until submodule commit is removed

Looks like that was just 63b9552

**Review by SweptWasTaken — APPROVED** [review]

**Comment by SethLafuente** [comment]
ok

**Comment by SethLafuente** [comment]
whoever is helping mw add more stuff and organize thanks but there is a bunch of errors poping in and I have no idea whats going anymore reveal yourself

**Comment by PaulRitter** [comment]
> whoever is helping mw add more stuff and organize thanks but there is a bunch of errors poping in and I have no idea whats going anymore reveal yourself

it is literally impossible for someone to do this. i'd urge you to read the git guide i sent you in discord

**Comment by SethLafuente** [comment]
changed everything so it would work with the current version of the game even Capitalized all the materials on stuff there should be no errors

File: Resources/Prototypes/Recipes/Lathes/cooking.yml [hunk 1]
```diff
@@ -4,36 +4,38 @@
...
-    steel: 300
+    Steel: 300
 
 - type: latheRecipe
   id: KitchenKnife
-  icon: Objects/Weapons/Melee/kitchen_knife.rsi
-  result: KitchenKnife
-  completetime: 400
+  icon:
+    sprite: Objects/Weapons/Melee/Kitchen_knife.rsi
```
Comment (mirrorcult) [inline Resources/Prototypes/Recipes/Lathes/cooking.yml:12]: should be kitchen_knife.rsi, no?

File: Resources/Prototypes/Recipes/Lathes/tools.yml [hunk 2]
```diff
@@ -112,5 +112,23 @@
...
+  id: Mop
+  icon:
+    sprite: Objects/Specific/Janitorial/mop.rsi
+    state: mop
+  result: MopItem
+  completetime: 500
+  materials:
+    Steel: 200
+    Plastic: 100
+
```
Comment (mirrorcult) [inline Resources/Prototypes/Recipes/Lathes/tools.yml:128]: remove these extra empty lines at the end of the file

Comment (SethLafuente) [reply Resources/Prototypes/Recipes/Lathes/tools.yml:128]: ok

File: Resources/Prototypes/Recipes/Lathes/cooking.yml (hunk 1 above)
Comment (SethLafuente) [reply Resources/Prototypes/Recipes/Lathes/cooking.yml:12]: is Kitchen_knife.rsi It works like that so yeah

File: Resources/Prototypes/Catalog/Research/technologies.yml [hunk 3]
```diff
@@ -27,6 +27,10 @@
   unlockedRecipes:
     - Scythe
     - Hatchet
+    - KitchenKnife
```
[the remark this answers is not in the document]
Comment (SethLafuente) [reply Resources/Prototypes/Catalog/Research/technologies.yml:30]: There is no other technology that can fit this and im still not to sure how to add new technologies I see no harm on those staying there for now I will learn and add more technologies later on

[the remark this answers is not in the document]
Comment (SethLafuente) [reply Resources/Prototypes/Catalog/Research/technologies.yml:30]: I learned how to add new technologies I added it putted the mop and bucket in their and putted the fire extinguisher in enginiring technology like a developer told me to 
image
it works just fine

**Comment by metalgearsloth** [comment]
Mark this pr as ready for review when you stop changing it

**Comment by SethLafuente** [comment]
> Mark this pr as ready for review when you stop changing it

I already stopped editing about 1 hour ago I’m done for the night eating cereal

File: Resources/Prototypes/Recipes/Lathes/cooking.yml (hunk 1 above)
Comment (SweptWasTaken) [reply Resources/Prototypes/Recipes/Lathes/cooking.yml:14]: This works for some reason but it really shouldn't.

**Comment by SweptWasTaken** [comment]
You committed submodule again in a680354.

Also tests are failing.

You also need to fix your changelog. That's not a viable format and will not work. Additionally you do not need to report every tiny change in it or post so many screenshots like you did. It's excessive and unnecessary.

**Comment by SethLafuente** [comment]
> You committed submodule again in [a680354](https://github.com/space-wizards/space-station-14/commit/a680354c39507c0512e94371c0edab552166a12d).
> 
> Also tests are failing.
> 
> You also need to fix your changelog. That's not a viable format and will not work. Additionally you do not need to report every tiny change in it or post so many screenshots like you did. It's excessive and unnecessary.


All the screenshoots are every single thing I added I could mergedem all into one

File: Resources/Prototypes/Recipes/Lathes/cooking.yml (hunk 1 above)
Comment (SethLafuente) [reply Resources/Prototypes/Recipes/Lathes/cooking.yml:14]: why shouldn't it work thats the name of the file?

**Comment by SethLafuente** [comment]
I fixed every issue and everything looks good in my part

**Comment by ShadowCommander** [comment]
You're going to need to remove the RobustToolbox changes from the PR so that tests can run.
```
git checkout master -- RobustToolbox
```
then commit and push that.

**Comment by SethLafuente** [comment]
> You're going to need to remove the RobustToolbox changes from the PR so that tests can run.
> 
> ```
> git checkout master -- RobustToolbox
> ```
> 
> then commit and push that.

Did it it says the branch is up to date and it dosent let me commit anything after putting in hat command

**Comment by SethLafuente** [comment]
I think it works now?
image

**Comment by SweptWasTaken** [comment]
Your changelog is still not in the correct format even after you changed it so I updated it for you. Please read the PR template next time as you obviously didn't.

**Comment by SethLafuente** [comment]
> Your changelog is still not in the correct format even after you changed it so I updated it for you. Please read the PR template next time as you obviously didn't.

No I did added sutff correctly I have no idea why its not showing

**Comment by SethLafuente** [comment]
Gona submit another pull request using another branch without is weird error

**Review by SweptWasTaken — APPROVED** [review]

**Pull request merged**